oranda-css/css/components.css

```diff
@@ -25,5 +25,5 @@ footer {
 }
 
 .repo_banner > a {
-  @apply flex justify-center gap-2 items-start text-slate-50 dark:text-axo-black h-[20px];
+  @apply flex justify-center gap-2 items-start hover:text-slate-50 text-slate-50 dark:text-axo-black dark:hover:text-axo-black h-[20px];
```

Comment:
lol i hate how git shows this diff hahaha

what's the behavior here? i do think on hover it should do something- maybe an underline? i can't tell exactly what this does but my sense is that it just keeps the text black, which is better than it disappearing, but probably not all we want to happen